Catalyst.Fabric.Authorization.Client/AuthorizationClient.cs

```diff
@@ -340,9 +392,33 @@ private void CheckIfNull(object value, string name)
             }
         }
 
-        private async Task<T> SendAndParseJson<T>(HttpRequestMessage message)
+        private async Task<T> SendAndParseJson<T>(HttpRequestMessage message, CancellationToken? token)
+        {
+            // This function wraps all the retry/circuit breaker logic
+            // for the sending of a request.
+            
+            // retry first a few times.
+            var RetryTask = RetryPolicy.ExecuteAsync<T>(

```

Comment:
Should we use the PolicyWrap policy so the circuit breaker can be nested inside the retry?